README.md

```diff
@@ -279,6 +279,10 @@ OPTIONS
   -t, --templates=templates
       [default: templates] Path to the templates folder
 
+  --che-operator-cr-patch=che-operator-cr-patch
```

Comment:
well, it's actually checluster-cr... Maybe it makes sense to name it like this then?